src/vehicle.cpp

```diff
@@ -178,7 +177,7 @@ void vehicle::add_missing_frames()
         std::pair<int, int> mount_location = std::make_pair( next_x, next_y );
 
         if( locations_checked.count( mount_location ) == 0 ) {
-            std::vector<int> parts_here = parts_at_relative( next_x, next_y, false );
+            std::vector<int> parts_here = parts_at_relative( point( next_x, next_y ), false );
```

Comment:
Why not keep the old version of parts_at_relative() for instances like this?

Reply:
Because I intend to remove things like this. There should be nearly no access of the x,y,z values of a point. Operations like applying an offset should be done directly as `some_point + some_vector` with both operands being `point` or `tripoint`.